components/prism-nevod.js

```diff
@@ -0,0 +1,79 @@
+Prism.languages.nevod = {
+	'comment': /(?:\/\/.*)|(?:\/\*[\s\S]*?(?:\*\/|$))/,
+	'string': [
+		{
+			pattern: /"(?:""|[^"])*"(?!")!?\*?/,
+			greedy: true,
+			inside: {
+				'modifiers': /!$|!\*$|\*$/,
+			},
+		},
+		{
+			pattern: /'(?:''|[^'])*'(?!')!?\*?/,
+			greedy: true,
+			inside: {
+				'modifiers': /!$|!\*$|\*$/,
+			},
+		}
+	],
+	'keyword': /@(?:inside|outside|having|search|where)\b/,
+	'require': /@require\b/,
+	'namespace': {
+		pattern: /@namespace\s+[a-zA-Z0-9\-.]+\s*{/,
+		inside: {
+			'keyword': /@namespace\b/,
+			'name': /\b[a-zA-Z0-9\-.]+/
+		},
+	},
+	'pattern': {
+		pattern: /(?:@pattern\s+)?#?[a-zA-Z0-9\-.]+\s*(?:(?:[(]\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]+\s*(?:,\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]*))*)[)])\s*)?=/,
+		inside: {
+			'keyword': /@pattern\b/,
+			'name': {
+				pattern: /^(\s+)?#?[a-zA-Z0-9\-.]+/,
+				lookbehind: true
+			},
+			'equal': /=/,
+			'attributes': {
+				pattern: /\(.*\)/,
+				inside: {
+					'attr-name': /[a-zA-Z0-9\-.]+/,
+					'attr-hidden-mark': /~/,
+					'attr-punctuation': /[,]/,
+					'attr-bracket': /[()]/,
+				},
+			},
+		},
+	},
+	'search': {
+		pattern: /(?:@search\s+|#)[a-zA-Z0-9\-.]+(?:\.\*)?\s*(?=;)/,
+		inside: {
+			'keyword': /@search\b/,
+			'name': /#?.*(?=;)/
+		}
+	},
+	'basic-reference': {
+		pattern: /\b(?:Word|Punct|Symbol|Space|LineBreak|Start|End|Alpha|AlphaNum|Num|NumAlpha|Blank|WordBreak|Any)(?:\(.*\)|\b)/,
```

Comment:
I'm not quite sure about the `\(.*\)`. It seems to be a little too simple to work. Those are token parameters, right? Should the parameters not be tokenized (e.g. `Alpha(2-10, TitleCase)`)? Wouldn't parameters automatically get sensible highlighting if we didn't include them here?

Reply:
I have refined the pattern to detect the built-in pattern with the parameters.